.changeset/fuzzy-tips-clap.md

```diff
@@ -0,0 +1,6 @@
+---
+"@medusajs/modules-sdk": patch
+"@medusajs/types": patch
```

Comment:
todo: missing medusa package

Reply:
the resolve type did change to accept ModuleExports now, shouldn't this be released?